src/pages/index.js

```diff
@@ -1,13 +1,66 @@
-import Form from '../components/Form/Form';
-import Headline1 from '../components/Headline/Headline1';
-import Navigation from '../components/Navigation/Navigation';
+import dynamic from 'next/dynamic';
+import Link from 'next/link';
+import {useRouter} from 'next/router';
+import styled from 'styled-components';
 
-export default function HomePage() {
+export default function StartPage() {
+	const DynamicWrapper = dynamic(() => import('../components/styledWrapper'), {
```

Comment:
If you don't mind the hassle, try removing the dynamic wrapper and see if you really need it. 
I'm using `useStore()` without `persist()` as of now, but turned out I didn't need dynamic wrapper. 

Food for thought.

Reply:
Thanks for the hint :)

I tried without as I did not think about it in the beginning, and I got the Hydration error...
After implementation of the DynamicWrapper it was gone